src/plugins/mer/merdeploysteps.cpp

```diff
@@ -123,6 +123,26 @@ class MerPrepareTargetStepConfigWidget : public BuildStepConfigWidget
     }
 };
 
+class ElidedWebsiteLabel : public QLabel
+{
+public:
+    explicit ElidedWebsiteLabel(const QString &text, QWidget *parent=nullptr, Qt::WindowFlags f=Qt::WindowFlags()) :
+        QLabel("", parent, f)
+    {
+        m_text = text;
+    }
```

Comment:
if we never pass flags... style

```suggestion
    explicit ElidedWebsiteLabel(const QString &text, QWidget *parent = nullptr)
        : QLabel(QString(), parent)
        , m_text(text)
    {
    }

```